src/Parser.php

```diff
@@ -14,8 +14,13 @@ public function parse(string $filename)
         foreach ($lines as $line) {
             $line = trim($line);
             if ($line !== '') {
-                [$url, $status] = preg_split('/[\s]+/', $line);
-                $domains[] = new Domain($url, $status);
+                [$url, $status, $content] = preg_split('/[\s]+/', $line);
```

Comment:
This throws errors if we don't have 3 parts, but due to not showing them in `bin/cigar` it's never seen 👀